app/views/_includes/application/reference.njk

```diff
@@ -17,6 +17,40 @@
     }) }}
   {% endif %}
 {% endset %}
+{% set relationshipHtml %}
+  <p class="govuk-body">{{ "“" + reference.relationship.summary + "”" | nl2br }}</p>
+  {% if reference.relationship.validated %}
+    <p class="govuk-body">{{ appIcon({
+      classes: "govuk-!-margin-right-1",
+      icon: "tick",
+      hidden: true
+    }) }} Confirmed by the referee.</p>
+  {% else %}
+    <p class="govuk-body">Amended by the referee:</p>
+    {{ govukInsetText({
+      text: reference.relationship.correction,
+      classes: "govuk-!-margin-top-0 govuk-!-padding-top-0 govuk-!-padding-bottom-0"
+    }) }}
+  {% endif %}
+{% endset %}
+{% set safeguardingHtml -%}
+  {%- if reference.safeguarding.response == "yes" -%}
+    {{ govukInsetText({
+      html: reference.safeguarding.concerns | nl2br,
+      classes: "govuk-!-margin-top-0 govuk-!-padding-top-0 govuk-!-padding-bottom-0"
+    }) }}
+  {%- elif reference.safeguarding.response == "no" -%}
+    <p class="govuk-body">Referee has none</p>
```

Comment:
```suggestion
    <p class="govuk-body">Referee has no concerns</p>
```

Reply:
Again here I'd suggest:

Does the referee have concerns about safeguarding? Yes
What are the referee's concerns? [Blah blah blah]